modules/issue_tracker/templates/issue_tracker_comment_history.tpl

```diff
@@ -0,0 +1,8 @@
+<p></p>
+{foreach $commentHistory as $comment }
+    {if $comment.fieldChanged == 'comment'}
+        <p> [{$comment.dateAdded}] {$comment.addedBy} commented <i> {$comment.newValue} </i> </p>
+    {else}
+        <p> [{$comment.dateAdded}] updated the <b>{$comment.fieldChanged} </b> to {$comment.newValue} </p>
```

Comment:
Add {$comment.addedBy} here - all changes should show who made them (not just comment updates; e.g. when a user changes ticket status without commenting)